src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/AES/AesCipherTests.cs

```diff
@@ -927,18 +939,32 @@ public static void VerifyKnownTransform_CFB128_256_NoPadding_1_Extended()
                 feedbackSize: 128);
         }
 
-        [Fact]
-        public static void AesZeroPad()
+        [Theory]
+        [InlineData(CipherMode.CBC)]
+        [InlineData(CipherMode.CFB)]
+        public static void AesZeroPad(CipherMode cipherMode)
         {
             byte[] decryptedBytes;
             byte[] expectedAnswer;
 
             using (Aes aes = AesFactory.Create())
             {
+                aes.Mode = cipherMode;
                 aes.Padding = PaddingMode.Zeros;
 
-                int blockBytes = aes.BlockSize / 8;
-                int missingBytes = blockBytes - (s_multiBlockBytes.Length % blockBytes);
+                int alignBytes;
+
+                if (cipherMode == CipherMode.CFB)
+                {
+                    aes.FeedbackSize = 128;
+                    alignBytes = aes.FeedbackSize / 8;
+                }
+                else
+                {
+                    alignBytes = aes.BlockSize / 8;
+                }
+
+                int missingBytes = alignBytes - (s_multiBlockBytes.Length % alignBytes);
```

Comment:
Since this test is using the block size as the feedback size, wouldn't it be simpler to just set `aes.FeedbackSize = 128` regardless of the mode up where the padding mode and block mode are set?

Reply:
Yes, yes it would.